spec/spec_helper_acceptance.rb

```diff
@@ -29,13 +29,13 @@
       unless ENV['BEAKER_provision'] == 'no'
         shell("/bin/touch #{default['puppetpath']}/hiera.yaml")
         # install iis
-        on(agents, puppet("module install puppet/windowsfeature"))
+        on(agents, puppet("module install puppetlabs/dism"))
         pp=<<-EOS
-    $iis_features = ['Web-WebServer','Web-Scripting-Tools']
-
-    windowsfeature { $iis_features:
-      ensure => present,
-    }
+          dism { 'IIS-WebServer':
+            ensure    => present,
+            norestart => true,
+            all       => true
```

Comment:
While this will install the tools, it's overkill. We don't need the extra features, and it will add extra time to the tests